spec/features/spree/admin/reporting_to_taxjar_spec.rb

```diff
@@ -46,9 +44,22 @@
       end
 
       within("#order_tab_summary") do
+        expect(page).to have_text("Reported to TaxJar at: -", normalize_ws: true)
         expect(page).to have_text("TaxJar Sync: Error", normalize_ws: true)
-        expect(page).to have_text("Retry")
+
+        click_on "TaxJar Sync History"
+      end
+
+      within("#content-header") do
+        expect(page).to have_content("TaxJar Sync History")
       end
+
+      within('#transaction_sync_logs') do
+        expect(page).to have_content("Retry")
+        click_on "Retry"
+      end
+
+      expect(page).to have_content("Queued transaction sync job")
```

Comment:
Should we test that the sync succeeded here, like we do in the test above - https://github.com/SuperGoodSoft/solidus_taxjar/pull/244/commits/1fccf8417fe3ffe02d37acbaff40bbd980f397b8 🤔

Reply:
We actually don't update the thing that made the order invalid, so that's the reason we don't make that assertion.

We already know that successful sync logs will show up in the table if received.

So I think this coverage is good enough.